### PR TITLE
female->XX, male->XY

### PR DESCRIPTION
**Pull request by mike-w-wilson**
Small change to make this script compatible with the seqr sex_check.py script

### CODE REVIEW

File: tgg/relatedness/get_ibd.py [hunk 1]
```diff
@@ -233,8 +233,8 @@ def check_sex(sex_ht: hl.Table, output_dir: str, output_name: str,) -> None:
 
     ped_ht = ped_ht.annotate(
         given_sex=hl.case()
-        .when(ped_ht.Sex == "M", "male")
-        .when(ped_ht.Sex == "F", "female")
+        .when(ped_ht.Sex == "M", "XY")
+        .when(ped_ht.Sex == "F", "XX")
         .default(ped_ht.Sex)
     ).drop("Sex")
```
Comment (ch-kr) [inline tgg/relatedness/get_ibd.py:239]: why not make this annotate/drop a transmute?

**Review by ch-kr — DISMISSED** [review]
one small suggestion, but LGTM!

**Comment by mike-w-wilson** [comment]
I forgot to turn my autoformat on save off for this so it ran and dismissed the review, sorry @ch-kr !

**Review by ch-kr — APPROVED** [review]
thanks for reformatting!

**Pull request merged**